WinUIGallery/ControlPages/XamlFundamentals/ResourceDictionaryPage.xaml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Page x:Class="WinUIGallery.ControlPages.ResourceDictionaryPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:WinUIGallery"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <ResourceDictionary>
+            <x:Double x:Key="HeaderFontSize">24</x:Double>
+            <Color x:Key="AccentColor">#4CAF50</Color>
+            <SolidColorBrush x:Key="AccentBrush"
+                             Color="{StaticResource AccentColor}" />
+            <x:String x:Key="AppTitle">WinUI 3 Gallery</x:String>
+
+            <x:String x:Key="WelcomeMessage">Welcome to this page</x:String>
+            <SolidColorBrush x:Key="PageBackgroundBrush"
+                             Color="#E8EAF6" />
```

Comment:
Be very, very careful to hardcode colors like this. Did you check readability in light and dark mode? Is there no resource to use here?

Reply:
My goal was to demonstrate how to create resources and resource dictionaries in different ways. Using built-in resources wouldn't make sense in this context, as the purpose is to showcase the process of defining new resources rather than reusing existing ones.
However, if you see a better approach, your input is always welcome! :)

Reply:
Of course, makes complete sense, just needed a second look at this!